libethcore/EVMSchedule.cpp

```diff
@@ -0,0 +1,21 @@
+/// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+
+#include "EVMSchedule.h"
+#include <libethcore/ChainOperationParams.h>
+
+namespace dev
+{
+namespace eth
+{
+EVMSchedule addEIPsToSchedule(EVMSchedule const& _schedule, AdditionalEIPs const& _eips)
```

Comment:
Why don't we modify `_schedule` instead of creating a new `EVMSchedule `instance?

Reply:
Can't modify becasue fork schedules are constants. I can make it a static method or maybe a new constrctor of `EVMSchedule`